tests/Oryx.Integration.Tests/Python/PythonDynamicInstallationTest.cs

```diff
@@ -46,33 +48,37 @@ public async Task CanBuildAndRunPython39AppAsync()
         {
             // Temporarily skip - Bug #1266781
             // await CanBuildAndRunPythonAppAsync("3.9", ImageTestHelperConstants.GitHubActionsBuster);
+            await Task.FromResult(true);
         }
 
         [Fact]
         [Trait("category", "python-3.7")]
         [Trait("build-image", "github-actions-debian-bullseye")]
         public async Task CanBuildAndRunPython37App_UsingGitHubActionsBuildImage_AndDynamicRuntimeInstallationAsync()
         {
-            // Temporarily skip - Bug #1410367
-            // await CanBuildAndRunPythonApp_UsingGitHubActionsBuildImage_AndDynamicRuntimeInstallationAsync("PythonVersions.Python37Version", ImageTestHelperConstants.GitHubActionsBullseye);
+            // Temporarily skip - Bug #1266781
+            // await CanBuildAndRunPythonApp_UsingGitHubActionsBuildImage_AndDynamicRuntimeInstallationAsync(PythonVersions.Python37Version, ImageTestHelperConstants.GitHubActionsBullseye);
+            await Task.FromResult(true);
```

Comment:
Nit: would it be better to use Task.CompletedTask instead as this method is just expecting to return a Task? Not a big deal though, so feel free to ignore.

Reply:
I added this because we were receiving warnings that we were not using an `await` in the async function. So just returning the Task.CompletedTask wouldn't satisfy that requirement

Reply:
OK thanks for clarifying